### ISSUE TITLE
[Feature] Add a Playback Order to Play a TV Shows In Its Entirety Before Moving to the Next Show in a Collection

### ISSUE THREAD

**Issue by sxales** [ErsatzTV/ErsatzTV#1635]
For example, I have TV Show A that ran 3 seasons (2000, 2001, and 2002) and TV Show B that ran 1 (2001). 

Current Behavior: if I select chronological it will start playing A, then by the 2nd season it will alternate between A and B, until season 3 when it goes back to just playing A (i.e. AAAAAABABABABABAAAAA).

New Behavior: I would like a mode where Ersatz plays A in its entirety, then play B in its entirety (i.e. AAAAAAAAAABBBBBB).

Reason for the behavior: I am aware of a few franchises that have multiple TV series so it would make sense to group them, but the series have no continuity with one another so it doesn't make sense to alternate episodes.

**Comment by liam8888999** [comment]
try multiple of 0 which used to work but i havent tried it for a while

**Comment by sxales** [comment]
@liam8888999 Thanks for the suggestion, but that didn't work. It also flooded the channel with only episodes from that collection, ignoring the rest of the schedule.

**Comment by liam8888999** [comment]
@sxales sorry about that, it used to work but i guess the functionality has changed

**Comment by sxales** [comment]
@liam8888999 Yep, I changed every item in the schedule to multiple 0, and it just fills the playout with the first item marked multiple 0. Even then, if it was a collection, it still alternated episodes where they overlapped chronologically. It would be nice not have to manually update the schedule every few days.

**Comment by wdstudios2** [comment]
I'm confused as to why you'd put multiple TV shows in one collection to begin with.

**Comment by sxales** [comment]
@wdstudios2 How else would you automatically cycle through related series? If you manually update the schedule, it resets the play order.

**Comment by jasongdove** [comment]
If you are okay with the shows playing in a fixed order, this can be achieved using a playlist. Add all shows to the playlist and set each show to `play all` in the playlist.

The playlist can then be used as a schedule item that will not take over the entire schedule; you can use multiple, duration, etc.

**Comment by embolon** [comment]
This request looks similar to issue #1190
